packages/catalog-model/src/kinds/ComponentEntityV1alpha1.test.ts

```diff
@@ -103,6 +103,21 @@ describe('ComponentV1alpha1Validator', () => {
     await expect(validator.check(entity)).rejects.toThrow(/owner/);
   });
 
+  it('accepts missing subcomponentOf', async () => {
```

Comment:
you should add the field to the actual entity at the top too, to make sure that the "base case" works

Reply:
🤦